designer/pydesigner.py

```diff
@@ -718,6 +718,23 @@
             dp.writeNii(kfa, img.hdr, op.join(metricpath, 'kfa'))
             dp.writeNii(mkt, img.hdr, op.join(metricpath, 'mkt'))
             dp.writeNii(trace, img.hdr, op.join(metricpath, 'trace'))
+            if args.wmti:
```

Comment:
Should we make a separate path for WMTI?

Reply:
I think the `wmti-` prefix is good enough. This is a DKI-based metric after all that won't appear if users don't want it. We won't be enabling this by default.